### PR TITLE
docs: fix core team links in section in README

### LINKED ISSUE #2047: Links to commits by users still use older link in README.md
The [Core Team section](https://github.com/vuejs/vuepress#core-team) in the README.md is using the original Ulivz's repo address in the links for commits by the members.

> `https://github.com/Ulivz/VuePress/commits?author=ulivz`
> `..................|______________|`
> This redirects to a 404 page.

This should be using `.../vuejs/vuepress/...` now.

- [x] I confirm that this is an issue rather than a question.

#### Steps to reproduce

Try clicking on the emoji links under the member's name in [this section](https://github.com/vuejs/vuepress#core-team).

#### What is expected?

It should open `https://github.com/vuejs/vuepress/commits?author=[NAME]`

#### What is actually happening?

We get a 404 error page.

### PR DESCRIPTION
**Pull request by AjitZero**
**Summary**
Fixes #2047 

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,22 +60,22 @@ If you have a VuePress-related project/component/tool, add it with a pull reques
...
-    <td align="center"><a href="http://www.bencodezen.io"><img src="https://avatars0.githubusercontent.com/u/4836334?v=4" width="100px;" alt="Ben Hong"/><br /><sub><b>Ben Hong</b></sub></a><br /><a href="https://github.com/Ulivz/VuePress/commits?author=bencodezen" title="Code">💻</a> <a href="#blog-bencodezen" title="Blogposts">📝</a></td>
-    <td align="center"><a href="https://twitter.com/vicbergquist"><img src="https://avatars0.githubusercontent.com/u/25737281?v=4" width="100px;" alt="Victoria Bergquist"/><br /><sub><b>Victoria Bergquist</b></sub></a><br /><a href="https://github.com/Ulivz/VuePress/commits?author=vicbergquist" title="Code">💻</a> <a href="#design-vicbergquist" title="Design">🎨</a></td>
+    <td align="center"><a href="http://evanyou.me"><img src="https://avatars1.githubusercontent.com/u/499550?v=4" width="100px;" alt="Evan You"/><br /><sub><b>Evan You</b></sub></a><br /><a href="https://github.com/vue/vuepress/commits?author=yyx990803" title="Code">💻</a></td>
```
Comment (SigureMo) [inline README.md]: Should these `Ulivz`  be changed to `vuejs` instead of `vue`? :smile:

Comment (AjitZero) [reply README.md]: 🤦‍♂ Fixing this immediately.

**Comment by AjitZero** [comment]
Fixed the links again, from `/vue` to `/vuejs`, but noticed a different sort of issue.

The users `@filrak`, `@fatihacet` and `@farcaller` don't have any commits in this repo. All these GitHub accounts exist, so I'm not sure what's happening.

**Comment by SigureMo** [comment]
@AjitZero Sorry, I didn't read the `README.md` carefully, it seems auto generated by [All Contributors](https://allcontributors.org/docs/en/cli/configuration)(see [README.md line58](https://github.com/vuejs/vuepress/blame/master/README.md#L58)), so you change the `.all-contributorsrc` as well. :smile:

**Comment by AjitZero** [comment]
@SigureMo I've updated the `.all-contributorsrc` file, that should be it. I think the `README.md` changes need to dropped now.

**Pull request merged**